ChirpWiFiShareAndroid/app/src/main/java/io/chirp/chirpwifi/MainActivity.java

```diff
@@ -46,66 +46,49 @@ protected void onCreate(Bundle savedInstanceState) {
         status = findViewById(R.id.status);
         startStopListeningBtn = findViewById(R.id.startStopListening);
         context = this;
-        startStopListeningBtn.setAlpha(.4f);
-        startStopListeningBtn.setClickable(false);
         /*
-        You can download config string and credentials from your admin panel at developers.chirp.io
+        You can download config string and credentials from the developer portal at developers.chirp.io
          */
         String KEY = "YOUR_APP_KEY";
```

Comment:
Just one comment here. The copy and paste from the developer site now names these variables as `CHIRP_APP_KEY` etc, so we should stick to those so it's easily copy and pastable